package-scripts/datadog-agent/postinst

```diff
@@ -5,6 +5,8 @@ LOG_DIR=/var/log/datadog
 
 LINUX_DISTRIBUTION=$(grep -Eo "(Debian|Ubuntu|RedHat|CentOS|openSUSE|Amazon)" /etc/issue)
 
+chmod 755 /etc/init.d/datadog-agent
```

Comment:
Can you add a comment here, so that we remember why this instruction needs to be run first.